create-upgrade-symlinks.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+prev_versions=$(cat promscale.control | sed -n 's/# upgradeable_from = \(.*\)/\1/p' | sed "s/[[:space:]']//g" | tr ',' '\n')
```

Comment:
Does this mean we need to manually list all the `prev_versions` in that file?

Reply:
yes

Reply:
Note: this might change, depending on what exactly we do here: https://github.com/timescale/promscale_extension/issues/113